homeassistant/components/lyric/lyric/__init__.py

```diff
@@ -0,0 +1,1260 @@
+#  -*- coding:utf-8 -*-
```

Comment:
I'm open to suggestions on this.  The original library is this: https://github.com/bramkragten/python-lyric which is published to pypi as "python-lyric".  However, the version in pypi is missing key bug fixes from the repo, and the repo and pypi version appear to be abandoned in spite of my efforts to contact the original author.  In addition, I have added new functionality (fan mode management) that is missing from both versions. I'm hesitant to publish a python-lyric2  to pypi for a single project's use due to the confusion, that will likely cause and the fact that roughly 90% of that library would be the original author's work.  Please let me know your thoughts on this.

Reply:
Fixes should be submitted to the 3rd party library repo. I doubt that it's impossible to contact the author since he's employed by Nabu Casa and actively working on home assistant.

CC @bramkragten

Reply:
https://github.com/bramkragten/python-lyric/pull/5

Reply:
Will check it later this week

Reply:
@bramkragten: Thank you very much.  The changes are merged and pypi is updated.  I will incorporate into this branch and and resubmit when I get a chance, may be Jan 20th, before I can do that.